subgraphs/compound-forks/.prettierrc.json

```diff
@@ -1 +1,8 @@
-{}
```

Comment:
i think both are fine as long as they are consistent. the current code is also formatted, just under the default prettier schema